server/src/main/java/org/elasticsearch/action/admin/indices/mapping/get/GetFieldMappingsResponse.java

```diff
@@ -61,23 +60,18 @@
 
     GetFieldMappingsResponse(StreamInput in) throws IOException {
         super(in);
-        int size = in.readVInt();
-        Map<String, Map<String, FieldMappingMetadata>> indexMapBuilder = new HashMap<>(size);
-        for (int i = 0; i < size; i++) {
-            String index = in.readString();
-            if (in.getVersion().before(Version.V_8_0_0)) {
-                int typesSize = in.readVInt();
+        mappings = unmodifiableMap(in.readMap(StreamInput::readString, i -> {
```

Comment:
Since `i` is typically an integer, maybe use `mapIn` or something like that?

Reply:
Makes sense, renamed :)